wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,178 @@
+
+Now Click on the Pencil Icon to edit. And it should redirect you to the Web Editor.
+
+![GitHub's Web Editor](8.JPG "web Editor Snapshot")
+
+If you want to preview your changes, click the "Preview Changes" Tab. Click "Edit File" to go back to Editing.
+
+After making your changes, scroll down and Fill the following fields to explain your edit to other contributors.
+
+![Commit Message](9.JPG "Commit Message")
+
+You would be given a option where to save your edits.
+
+- "Commit Directly to the ``xy`` branch" will commit the changes through your branch directly.
+
+- "Create a New Branch" will automatically create a Branch for your edits. This is useful if you're making bulk edits.
+
+Pick a option and click "Commit Changes".
+
+### Through your local Machine
+
+GitHub under the hood is a Git Service. You can clone your fork in your Machine and make changes there.
+
+if you have git installed, put this in your Command Line Interface :
+```prolog
+git clone https://github.com/<username>/osu-wiki 
+```
+and this should clone your fork into your specified folder.
+
+Now after Making your changes locally, we need to commit this on your local copy.
+
+this can be done by 
+```prolog
+git add .
+git commit -m "<commit message here>"
+```
+However, in some Editors like Visual Studio Code, they provide a Git Management Interface.
+
+![Visual Studio code's Git Interface](10.JPG "Visual Studio's Git Interface")
+
+After Committing your changes, we ned to sync this on GitHub, to be able to do this, Enter the following in your Command Line Interface :
+
+```prolog
+git push origin <branch>
+```
+and this should upload your changes in GitHub.
+
+## Pull Requests
+
+After making your changes, you must make a "Pull Request" to be able for your changes to be made into the wiki.
+
+Go to your Pull Requests tab.
+
+![Pull Request Tab](6.JPG "Pull Request Tab")
+
+You should see the Pull Requests Page.
+
+![PR UI](11.jpg "Pull Requests Page")
+
+This is your Pull Request Page. Now click on "New Pull Request"
+
+and you should see something like this.
+
+![Compare Changes Page](12.jpg "Comparing Changes")
+
+Now we want to compare ``ppy:master`` against your changes.
+
+Now Select where did you made your changes in "branch" beside your "head Fork".
+
+![Comparing Changes 2](13.jpg "Comparing Changes 2")
+
+_However if you made changes in your Master Branch, The Changes will be shown without changing anything in the head fork options_.
+
+Now after viewing your changes, click "Create Pull Request" and fill it with the following details:
+
+**Title** : The Summary of the Changes you've made.
+
+**PR Sub-Message** : the Details of the changes you've made. 
+
+![PR Creation Dialogue](14.jpg "PR Creation Dialogue")
+
+Then Click "Create Pull Request" and wait for a Reviewer to check and possibly, Merge your changes.
+ 
+ Your Changes must follow the [Article Styling Guidelines](wiki/Article_Style_Guide/en.md) to be able to get merged.
```

Comment:
letter casing and remove `en.md` (this is handled automatically on the server side).... you might as well point out that the reviewers may help out in catching those errors of not following the article style guidelines.

`Your changes must follow the [Article Styling Guidelines](wiki/Article_Style_Guide/) to be merged. Nonetheless, don't fear this as that is why we review pull requests before we merging them!`